tools/lint-svg

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+const XMLDoc = require("xmldoc").XmlDocument;
+const chalk = require("chalk");
+const util = require("util");
+util.inspect.styles.string = "cyan";
+util.inspect.styles.regexp = "grey";
+const inspect = value => util.inspect(value, {colors: true});
+
+/**
+ * All the rules we use for SVG linting
+ * Are given a single argument, the parsed SVG object
+ * Should throw an error if it fails
+ */
+const RULES = [
+    function rootAttributes(svg) {
+        const expected = {
+            xmlns: "http://www.w3.org/2000/svg", 
+            viewBox: "0 0 24 24", 
+            "aria-labelledby": "title", 
+            role: "img"
+        };
+        const msgs = [];
+
+        for (let attr in expected) {
+            if (svg.attr[attr] !== expected[attr]) {
+                msgs.push(`SVG attribute ${chalk.bold(attr)} should be ${inspect(expected[attr])}, was ${inspect(svg.attr[attr])}`);
+            }
+        }
+        Object.keys(svg.attr)
+            .filter(attr => !expected.hasOwnProperty(attr))
+            .forEach(attr => {
+                msgs.push(`SVG attribute ${chalk.bold(attr)} should not exist`);
```

Comment:
The `stroke-*` attributes should definitely not exist, what with strokes being disallowed. The `fill-rule` and `clip-rule` do have possible applications, but they're pretty infrequent (and in case it's needed, you can always bypass the linter :))

I reworded it from "should not exist" to "is unexpected" just to make it a bit less harsh though. Good point